tst/EnergyPlus/unit/AirLoopHVACDOAS.unit.cc

```diff
@@ -4035,6 +4034,8 @@ TEST_F(EnergyPlusFixture, AirLoopHVACDOASTest)
 
     ASSERT_TRUE(process_idf(idf_objects));
 
+    state->init_state(*state);
```

Comment:
call `init_state()` after `process_idf()` to construct objects specified in the IDF file. For now, `init_state()` only constructs schedule and fluid properties objects, all other objects are still construced "on first use" by the ubiquitous `if (state.dataX->GetInput) { GetXInput(state); state.dataX->GetInput = false; }` pattern. Incrementally, more object types will be constructed this way.